android_build_files/android_abis.gradle

```diff
@@ -24,8 +24,9 @@ android {
       // Default list of ABIs available in up-to-date NDK.
       abiFilters "x86", "armeabi-v7a", "arm64-v8a", "x86_64"
 
-      if (System.getenv("ANDROID_NDK_HOME").contains("r16b") ||
-          System.getenv("ANDROID_NDK_HOME").contains("r11c")) {
+      if (System.getenv("ANDROID_NDK_HOME") != null && (
+              System.getenv("ANDROID_NDK_HOME").contains("r16b") ||
```

Comment:
Could you replace these tabs with spaces?

Reply:
It *was* spaces. Android Studio had formatted it like this. I tweaked it a bit. Let me know if you're okay with it now.